apps/rehash.c

```diff
@@ -562,6 +562,11 @@ int rehash_main(int argc, char **argv)
     } else if ((env = getenv(X509_get_default_cert_dir_env())) != NULL) {
         char lsc[2] = { LIST_SEPARATOR_CHAR, '\0' };
         m = OPENSSL_strdup(env);
+        if (m == NULL) {
+            BIO_puts(bio_err, "out of memory\n");
+            errs = -1;
```

Comment:
Since this seems to be the return value of main, the only valid values are EXIT_SUCCESS, EXIT_FAILURE and 0, other values are implementation defined. At least on Linux, only values from 0 to 255 are supported, and anything non-0 is an error. This code seems to set it to 1 or do ++ in other places. I suggest you also just set it to 1.

Reply:
Thank you for your kind reminder. I changed the return value to 1 in my new commit.